services/forms/admin.go

```diff
@@ -41,6 +41,7 @@ type AdminEditUserForm struct {
 	Password                string `binding:"MaxSize(255)"`
 	Website                 string `binding:"ValidUrl;MaxSize(255)"`
 	Location                string `binding:"MaxSize(50)"`
+	Language                string `binding:"MaxSize(5)"`
```

Comment:
MaxSize `5`?

Reply:
```suggestion
	Language                string `binding:"MaxSize(10)"`
```
even if we don't use it, give it a bit of space for the eventuality.

Reply:
we use the short identifier de-DE in the backend

Reply:
->
https://github.com/go-gitea/gitea/blob/e7cb8da2a8310ac167b6f613b283caa3316a7154/models/user/user.go#L94